src/ConsoleConfig.php

```diff
@@ -13,6 +13,7 @@ public function __construct(
 
         /** @var ConsoleCommand[] $commands */
         public array $commands = [],
+        public ?string $logPath = null,
```

Comment:
Configurable log path for tasks running in the background. If null, we'll use a sensible default.